### ISSUE TITLE
Automatic Seed Cluster Management

### ISSUE THREAD

**Issue by vlerenc** [gardener/gardener#2885]
# Stories
- As operator, I do not want to manually add or remove seeds, when load increases or decreases.
- As operator, I do not want to manually migrate control planes (depends on #1631), if a seed fails.

# Motivation
We lack seed automation that would deal with load or failure issues. Practically, we see people mostly hit by load issues, but there is also the risk of a failed seed. Today both these events can only be handled by highly experienced Gardener experts. This blocks any eventual adoption by less experienced, yet interested parties.

**Comment by timebertt** [comment]
/touch

Unfortunately, this topic (GEP-13) was put on hold in favor of other topics with higher priority, so no one is currently working on it.
/unassign @rfranzke @stoyanr @danielfoehrKn @timebertt 
/status on-hold
